modules/cnf-about-the-profile-creator-tool.adoc

```diff
@@ -6,11 +6,26 @@
 [id="cnf-about-the-profile-creator-tool_{context}"]
 = About the Performance Profile Creator
 
-The Performance Profile Creator (PPC) is a command-line tool, delivered with the Node Tuning Operator, used to create the performance profile.
-The tool consumes `must-gather` data from the cluster and several user-supplied profile arguments. The PPC generates a performance profile that is appropriate for your hardware and topology.
+The Performance Profile Creator (PPC) is a command-line tool, delivered with the Node Tuning Operator, which can help you to create a performance profile for your cluster.
 
-The tool is run by one of the following methods:
+PPC discovery mode::
```

Comment:
```suggestion

```

Reply:
Thanks, removed other instances of "discovery mode" in the steps so it only says "To display information about the cluster, run the PPC tool with the `log` argument..."